random_lsm.py

```diff
@@ -0,0 +1,24 @@
+import numpy.random as random
```

Comment:
so we actually don't need this file anymore right?

Reply:
Yes, you can delete it